init.ts

```
@@ -233,6 +233,13 @@ await Deno.writeTextFile(
   STATIC_LOGO,
 );
 
+const faviconArrayBuffer = await fetch("https://fresh.deno.dev/favicon.ico")
+  .then((d) => d.arrayBuffer());
+await Deno.writeFile(
+  join(directory, "static", "favicon.ico"),
+  new Uint8Array(faviconArrayBuffer),
+);
```

Could you wrap this in a try catch, so if there is a network error it will just silently ignore the favicon? Makes it a little more resilient to slow networks.

---

done! thanks